kevm-pyk/src/tests/integration/test_prove.py

```diff
@@ -210,25 +210,34 @@ def test_pyk_prove(
     spec_file: Path,
     kompiled_target_for: Callable[[Path], KompiledTarget],
     tmp_path: Path,
+    caplog: LogCaptureFixture,
 ) -> None:
+    caplog.set_level(logging.INFO)
+
     if spec_file in SKIPPED_PYK_TESTS:
         pytest.skip()
 
     # Given
+    log_file = tmp_path / 'log.txt'
```

Comment:
Can you define the log file's name somewhere instead of hardcoding it?

Reply:
It's not hard to implement that. For now, I think hardcoding the name is good enough as it is unique for the test run.